docs/modules/ROOT/pages/visual-tour.adoc

```diff
@@ -365,8 +370,8 @@ The example shows how to change the `maxFrames` from its default of `50` to `10`
 ----
 :config {maxFrames:10, theme: "outline"}
 ----
-Once making the change, rerunning `:config` should report that `maxFrames` has been set to `10` and `theme` is defined to `outline`.
-The Browser Settings Icon should also reflect this change.
+Once making the change, rerunning `:config` should report that `maxFrames` has been set to 10, and the theme is defined to outline.
+The *Browser Settings* drawer also reflects this change.
 ====
```

Comment:
```
[WARNING]
====
This will first reset the configuration to the default configuration and then set the given configurations.
====
```

Reply:
Why a WARNING sign?

Reply:
Its because the title is "Adjust several settings", so if you already set one e.g. `:config maxNeighbours:20` and then think that you can set several other settings after with this style e.g. `:config {maxFrames: 10, theme: "outline"}`, the setting `maxNeighbours` will be reset to the default value.